### DISCUSSION TITLE
Version 2.9 installation needs support

### DISCUSSION THREAD

**Discussion by AvinashKulkarniHyd** [egovernments/Digit-Core#270] [DIGIT Platform]
1.	Latest 2.9 version installation video support not available. YOUTUBE videos are old sept 2022 vintage. 
2.	DIGIT 2.9 build or earlier version is not stable. 2.7/2.4/2.9 versions we have tried to install but we don’t get all the modules installed properly. We can see the versions but with errors. PGR module comes up fine. We want to setup file for DEVOPS for all modules not the three that’s embedded in the setup file. 
3.	After the installation, language options are not enabled properly.

**Comment by naresh-egov** [comment]
Please use the following Chart to install the v2.9

https://github.com/egovernments/DIGIT-DevOps/blob/master/deploy-as-code/helm/product-release-charts/DIGIT/dependency_chart-digit-core-v2.9.yaml
